hack/doc/README.md

```diff
@@ -31,3 +33,25 @@ make doc/permissions
 python3 <operator-repo>/hack/doc/role-permissions2md.py https://github.com/Dynatrace/dynatrace-operator/releases/download/v0.12.0/openshift.yaml
 python3 <operator-repo>/hack/doc/role-permissions2md.py https://github.com/Dynatrace/dynatrace-operator/releases/download/v0.12.0/openshift-csi.yaml
 ```
+
+## custom_resource_params_to_md.py
+
+This script generates API docs for custom resources
+
+Pre-requisites:
```

Comment:
prerequisites are formatted differently in the previous script. Let's try to align them